mempool/clist_mempool.go

```diff
@@ -600,9 +600,7 @@ func (mem *CListMempool) GetNewTxs(peerID uint16, max int) (ret []*types.Tx) {
 		// Only add/return this if the peer hasn't seen it
 		memTx := next.Value.(*mempoolTx)
 
-		// Priority txs are gossiped using peer pointer priority txs so we to do not include
-		// them here to avoid sending duplicates
-		if _, ok := memTx.senders.Load(peerID); !ok && !isPriority(memTx.tx) {
+		if _, ok := memTx.senders.Load(peerID); !ok {
```

Comment:
Removed check for isPriority as sometimes a peer does not yet exist when DKG txs are added to mempool. Removing this check allows these txs to be included from the main mempool queue. Since DKG transactions are appended in the front of the queue there will not be many duplicates in the gossip beyond the first few transactions.

Reply:
Sure. Is this just fixing the corner case when the network starts for the first time?